docs/fundamentals/custom-resource-urls.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Define custom resource URLs
+description: Learn how to create custom URLs for .NET Aspire resources.
+ms.date: 04/08/2025
+ms.topic: how-to
+---
+
+# Define custom resource URLs
```

Comment:
One last thing to call out is that custom URLs only apply to "run" mode, not "publish" mode, i.e. it's only for configuring resource URLs that are displayed on the dashboard.